spec/support/shared_examples/password_strength.rb

```diff
@@ -94,4 +94,27 @@
     end
     expect(form.submit(password: password)).to eq result
   end
+
+  it 'does not allow a password that does not have the minimum number of graphemes' do
+    user = build_stubbed(:user, email: 'custom@benevolent.com', uuid: '123')
+    allow(user).to receive(:reset_password_period_valid?).and_return(true)
+    form = form_class.constantize.new(user)
+    password = 'a7K!hf🇺🇸🇺🇸🇺🇸'
+    errors = {
+      password: ['is too short (minimum is 12 characters)'],
```

Comment:
Could we anticipate a fix to the lack of translation for this string?

```suggestion
      password: [t('errors.messages.too_short', count: 12)],
```